fragments/home/cards.yml

```diff
@@ -29,7 +29,7 @@
     - url: /disability/upload-supporting-evidence/
       title: Upload evidence to support your claim
       nav_path: Upload evidence to support your claim
-    - url: /disability/how-to-file-claim/
+    - url: /disability-benefits/apply/form-526-disability-claim/introduction
```

Comment:
Generally, we should leave off the `introduction` part of the url here, but it will work either way.